### DISCUSSION TITLE
Custom column for things like Flow, Pressure Advance etc.

### DISCUSSION THREAD

**Discussion by Symmster** [Donkie/Spoolman#269] [Ideas]
Is it possible to create your own columns for specific filament properties? 
I would like to save things like Flow, Pressure Advance or Retraction Settings. Currently I use the comment function for this, but I would prefer to have each individual point as a column and not all values in one cell. 
In combination with a .csv export function based on the filament tab, this would be the optimal solution

**Comment by Donkie** [comment]
This is something I'm actively working on, and will be available in the next release.
